GentApp/Views/HomePage.xaml

```diff
@@ -2,18 +2,98 @@
     x:Class="GentApp.Views.HomePage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:GentApp.Views"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-	DataContext="{Binding CompaniesViewModelInstance, Source={StaticResource Locator}}">
+	xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+	xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+	xmlns:datamodel="using:GentApp.DataModel"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	DataContext="{Binding BranchesViewModelInstance, Source={StaticResource Locator}}">
 
-	<RelativePanel
-			HorizontalAlignment="Center"
-			VerticalAlignment="Center">
-		<TextBlock 
-			Text="Home"
-			FontSize="70"/>
-	</RelativePanel>
+	<interactivity:Interaction.Behaviors>
+		<core:EventTriggerBehavior EventName="Loaded">
+			<core:InvokeCommandAction Command="{Binding Path=LoadBranchesCommand}" />
+			<core:InvokeCommandAction Command="{Binding Path=LoadSubscriptionsCommand}" />
+		</core:EventTriggerBehavior>
+	</interactivity:Interaction.Behaviors>
+
+	<Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="*"></RowDefinition>
+			<RowDefinition Height="310"></RowDefinition>
+		</Grid.RowDefinitions>
+		<ParallaxView Grid.Row="0" HorizontalShift="400" Source="{x:Bind HomeFeedGrid}" VerticalAlignment="Bottom" Margin="0,-228,0,0">
+			<Image 
+                   Margin="-0,0,0,0"
+                   Stretch="UniformToFill"
+                   MinHeight="690"
+				Source="/Assets/ghent_parallax.png">
+			</Image>
+		</ParallaxView>
+		<StackPanel Orientation="Horizontal"
+					Margin="24,68,24,310"
+					Grid.Row="0" Grid.RowSpan="2">
+			<ComboBox x:Name="companyTypeComboBox"
+					  PlaceholderText="Filter by sort"
+					  Margin="2"/>
+			<AutoSuggestBox PlaceholderText="Search a company"
+							Width="200"
+							Margin="2"
+							QueryIcon="Find"/>
+		</StackPanel>
+		<TextBlock
+			Grid.Row="1" 
+			VerticalAlignment="Top" 
+			FontWeight="Medium" 
+			Margin="24,12,0,0" 
+			Text="Recommended" />
```

Comment:
Recommended

Reply:
kvind da nog iets hebben :p als er niets staat is het zo leeg precies

Reply:
ik vind da raar dat da daar staat

Reply:
bij netflix en youtube is da logisch da ze met recommended werken, want daar kijken ze naar uw vorige keuzes en waj al bekeken hebt en kunnen ze via algoritme dingen aanbevelen, maar wij gaan ni zo'n algoritme hebben eh, tenzij je wilt sorteren op subscribed ofzo

Reply:
dus WEG!!!!!!!

Reply:
okee